unittest/unittest

```diff
@@ -40,6 +40,14 @@ then
   haderror="Y"
 fi
 
+if [ ! -f ../sbin/install.config.in ]
```

Comment:
Not sure the unittests get invoked by make check at the moment but I will look into this.